corehq/apps/registration/forms.py

```diff
@@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
     """
     Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
     """
-    full_name = forms.CharField(label=_('Full Name'),
```

Comment:
Ha. I recently used `full_name` for interrupt https://github.com/dimagi/commcare-hq/pull/8109/files

Just realizing now that keeping the username kwarg was probably not the most clear idea